beacon_node/network/src/network_beacon_processor/rpc_methods.rs

```diff
@@ -425,17 +433,7 @@ impl<T: BeaconChainTypes> NetworkBeaconProcessor<T> {
         };
 
         // Pick out the required blocks, ignoring skip-slots.
-        let mut last_block_root = req
-            .start_slot()
-            .checked_sub(1)
-            .map(|prev_slot| {
-                self.chain
-                    .block_root_at_slot(Slot::new(prev_slot), WhenSlotSkipped::Prev)
-            })
-            .transpose()
-            .ok()
-            .flatten()
-            .flatten();
+        let mut last_block_root = None;
```

Comment:
This is how this exists in unstable. The more complicated `last_block_root` is necessary only for blobs, because we don't truncate responses by slot (unlike with blocks).